src/pages/patientView/mutationalSignatures/MutationalSignatureBarChartUtils.ts

```diff
@@ -0,0 +1,381 @@
+        subcategory: '2',
+        color: '#f1948a',
+    },
+    {
+        name: '2:Del:M',
+        category: 'Microhomology',
+        subcategory: '2',
+        color: '#D2B7F2',
+    },
+    {
+        name: '3:Del:R',
+        category: '>1bp deletion',
+        subcategory: '3',
+        color: '#ec7063',
+    },
+    {
+        name: '3:Del:M',
+        category: 'Microhomology',
+        subcategory: '3',
+        color: '#9b59b6',
+    },
+    {
+        name: '4:Del:R',
+        category: '>1bp deletion',
+        subcategory: '4',
+        color: '#e74c3c',
+    },
+    {
+        name: '4:Del:M',
+        category: 'Microhomology',
+        subcategory: '4',
+        color: '#7d3c98',
+    },
+    {
+        name: '5:Del:R',
+        category: '>1bp deletion',
+        subcategory: '5',
+        color: '#cb4335',
+    },
+    {
+        name: '5:Del:M',
+        category: 'Microhomology',
+        subcategory: '5',
+        color: '#4a235a',
+    },
+    {
+        name: '1:Ins:T',
+        category: '1bp insertion',
+        subcategory: 'T',
+        color: '#28b463',
+    },
+    {
+        name: '1:Ins:C',
+        category: '1bp insertion',
+        subcategory: 'C',
+        color: '#82e0aa',
+    },
+    {
+        name: '2:Ins:M',
+        category: 'Microhomology',
+        subcategory: '2',
+        color: '#aed6f1',
+    },
+    {
+        name: '2:Ins:R',
+        category: '>1bp insertion',
+        subcategory: '2',
+        color: '#33ffff',
+    },
+    {
+        name: '3:Ins:M',
+        category: 'Microhomology',
+        subcategory: '3',
+        color: '#85c1e9',
+    },
+    {
+        name: '3:Ins:R',
+        category: '>1bp insertion',
+        subcategory: '3',
+        color: '#aed6F1',
+    },
+    {
+        name: '4:Ins:M',
+        category: 'Microhomology',
+        subcategory: '4',
+        color: '#85c1e9',
+    },
+    {
+        name: '4:Ins:R',
+        category: '>1bp insertion',
+        subcategory: '4',
+        color: '#5dade2',
+    },
+    {
+        name: '5:Ins:M',
+        category: 'Microhomology',
+        subcategory: '5',
+        color: '#3498db',
+    },
+    {
+        name: '5:Ins:R',
+        category: '>1bp insertion',
+        subcategory: '5',
+        color: '#2874a6',
+    },
+];
+export function getColorsForSignatures(
+    dataset: IMutationalCounts[]
+): IColorLegend[] {
+    const colorTableData = dataset.map((obj: IMutationalCounts) => {
+        if (obj.mutationalSignatureLabel !== '') {
+            const colorIdentity = colorMap.filter(cmap => {
+                if (obj.mutationalSignatureLabel.match(cmap.name) !== null) {
+                    return cmap.color;
+                }
+            });
+            const label = obj.mutationalSignatureLabel;
+            const group: string =
+                colorIdentity.length > 0
+                    ? colorIdentity[colorIdentity.length - 1].category
+                    : 'unkown';
```

Comment:
hard-coding a string value is always prone to these kind of typos

```suggestion
                    : 'unknown';
```